draft-ietf-httpbis-priority.md

```diff
@@ -402,10 +402,13 @@ Prioritized Stream ID:
 Priority Field Value:
 : The priority update value in ASCII text, encoded using Structured Fields.
 
-The Prioritized Stream ID MUST be within the stream limit. If a
-server receives a PRIORITY_UPDATE with a Prioritized Stream ID that is beyond
-the stream limits, this SHOULD be treated as a connection error of type
-PROTOCOL_ERROR.
+The Prioritized Stream ID SHOULD be a value within the stream concurrency
+window. A server SHOULD NOT treat a Prioritized Stream ID beyond this window as
```

Comment:
This doesn't appear to be a term defined in RFC 7540.  I think I'd try to frame this in terms of the number of active streams ("open" state or either "half-closed" state) and the value of SETTINGS_MAX_CONCURRENT_STREAMS, both of which are defined for HTTP/2.

Reply:
Thanks for calling me up on this, its exactly the type of review that's needed.